samples/csharp_dotnetcore/51.cafe-bot/Dialogs/QnA/resources/qna.lu

```diff
@@ -0,0 +1,82 @@
+> QnA Pairs for cafe bot
+> Check out https://github.com/Microsoft/botbuilder-tools/tree/master/packages/Ludown to learn more about .lu files and Ludown.
+
+## ? What are your locations
+```markdown
+We have Cafe locations in Redmond, Seattle, Renton and Bellevue
+```
+
+## ? What are your store hours
+- store hours
+- timings
+```markdown
+Most cafe locations are open 8AM through 10PM local Pacific time. 
+```
+
+## ? who is your CEO
+- who owns contoso cafe
+```markdown
+Joshua Newmann is the CEO of Contoso Cafe
+```
+
+## ? Are you hiring
+- Do you have job openings?
+- are you looking for a barrista
+- can i come work for contoso cafe?
+```markdown
+You bet, we always are. Please visit http://contosocafe.com/careers
+```
+
+## ? Do you have Wifi?
+- Wifi access
+- wifi access at your stores? 
+- how about wifi access in your stores? 
+- do you offer free wifi?
+```markdown
+Absolutely. All our stores offer free wi-fi so you can stay connected and enjoy a cup of coffee! 
+```
+
+## ? Free parking?
+- do you offer parking? 
+- is free parking available? 
+- how about free parking at your stores? 
+```markdown
+Yes. All our stores offer free parking for guests.
+```
+
+## ? high chair?
+- do you have high chair for kids? 
+- can i bring my kids?
+- do you have high chairs?
+- is it ok to bring my son?
+- can i bring my daughter along?
+```markdown
+Guests all age are welcome at Contoso Cafe! We offer convenient seating for all age groups and can provide high chairs if needed. We are more than happy to accomodate your needs while you are here. Just let one of the staff know! 
```

Comment:
There's a typo here with `accomodate`, it should be `accommodate` with double `m`.

Reply:
fixed